nano/core_test/toml.cpp

```diff
@@ -73,7 +73,7 @@ TEST (toml, daemon_config_update_array)
 	std::filesystem::path data_path (".");
 	nano::daemon_config c{ data_path, nano::dev::network_params };
 	c.node.preconfigured_peers.push_back ("dev-peer.org");
-	c.serialize_toml (t);
+	c.serialize_toml ();
```

Comment:
This test does nothing now, because the returned toml string is discarded. This test doesn't make much sense IMO. I would just delete it